lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -107,8 +107,8 @@ def scrub(transcript)
           gsub(%r((\"card_number\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"security_code\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"emv_issuer_data\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cryptogram\\\":\\\"/)\w+), '\1[FILTERED]').
-          gsub(%r((\"token_card_data\\\":{\\\"token\\\":\\\")\d+), '\1[FILTERED]')
+          gsub(%r((\"cryptogram\\\":\\\")\w+), '\1[FILTERED]').
```

Comment:
Before this I was only taking the cryptograms that started with => `/` (slash)

Reply:
keep the slash because Decidir always returns one in the cyrptogram and this is the only way it'll be scrubbed properly

Reply:
In staging it returns a different cryptogram but I'm going to reverse the change correctly

Reply:
Done!